src/pages/NewChatPage.tsx

```diff
@@ -229,7 +229,7 @@ function NewChatPage({isGroupChat}: NewChatPageProps) {
 
     const itemRightSideComponent = useCallback(
         (item: ListItem & OptionsListUtils.Option, isFocused?: boolean) => {
-            if (!!item.isSelfDM || (item.accountID && CONST.NON_ADDABLE_ACCOUNT_IDS.includes(item.accountID))) {
+            if (!!item.isSelfDM || (item.login && excludedGroupEmails.includes(item.login))) {
```

Comment:
Can you add more context about using `login` instead of `accountID`? Is there any comparison between them?

Reply:
Yes, the excludedGroupEmails contains the logins of all Expensify accounts. That's why I'm using login here. Previously, we were using NON_ADDABLE_ACCOUNT_IDS, which had the account IDs for CHRONOS and NOTIFICATIONS. However, excludedGroupEmails also includes the logins for these two accounts.

Reply:
More context about using `login` instead of `accountID` is `accountID` can be `optimisticAccountID` in `userToInvite` util API respond with the full user data

https://github.com/Expensify/App/blob/652d2ffa1b099f9c5a1f0cdce780792097f736ad/src/libs/OptionsListUtils.ts#L1661-L1664

And this can cause this issue
1. Login with new account (to confirm there is no local data)
2. Click fab > start chat
3. Search for `chronos@expensify.com` for the first time

You will see `add to group` is displayed temporary, and you can add it to group util API respond with the full user data.


https://github.com/user-attachments/assets/00271afd-b071-4264-97c8-70920819bab8

Reply:
I think we need to remove `NON_ADDABLE_ACCOUNT_IDS` because it is not used in anywhere now